style.css

```diff
@@ -528,3 +577,550 @@ div.skills-title {
+  }
+
+  nav > ul {
+    display: flex;
+    gap: 5rem;
+  }
+
+  nav > span {
+    font-size: 1.2rem;
+  }
+
+  .hidden-items {
+    display: inline;
+    font-size: 1rem;
+  }
+
+  h1 {
+    font-size: 5rem;
+    padding-top: 12rem;
+  }
+
+  .H-title {
+    font-size: 2rem;
+    padding: 1.2rem 0 1.2rem 2.2rem;
+  }
+
+  .p {
+    font-size: 1.3rem;
+    line-height: 2.5rem;
+  }
+
+  .social-media {
+    position: relative;
+    top: -18.5rem;
+    left: 3rem;
+  }
+
+  .social-media > ul {
+    height: 12.5rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    width: 1.2rem;
+  }
+
+  section#projects {
+    display: grid;
+    grid-template-columns: 10rem 1fr 10rem;
+    grid-template-rows: 1fr 5fr;
+    padding: 0 0 10rem 0;
+  }
+
+  #h2-container {
+    grid-column: 2 / 3;
+    grid-row: 1 / 2;
+    margin: 6rem 0;
+    width: 100%;
+    display: flex;
+    align-items: center;
+  }
+
+  #h2-project {
+    border-bottom: none;
+    margin-top: 0;
+    width: 315px;
+  }
+
+  div#h2-line {
+    height: 1px;
+    width: 30%;
+    background-color: #212121;
+  }
+
+  #portfolio-grid {
+    grid-column: 2 / 3;
+    grid-row: 2 / -1;
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  #main-project {
+    grid-column: 1 / -1;
+    grid-row: 1 / 2;
+    display: flex;
+    align-items: flex-start;
+    gap: 7.2rem;
+  }
+
+  #project-works {
+    width: 684px;
+    height: 448px;
+  }
+
+  #project-h3 {
+    font-size: 2.5rem;
+    padding: 0 0 2rem 1rem;
+  }
+
+  #content-p {
+    font-size: 1.5rem;
+    line-height: 2rem;
+    width: 750px;
+  }
+
+  .language-button12 {
+    justify-content: flex-start;
+    gap: 0.8rem;
+    margin: 2.9rem 0 1.4rem 0;
+  }
+
+  .mini-projects {
+    height: 700px;
+    width: 100%;
+  }
+
+  div.papd-p1 {
+    padding-right: 2rem;
+    font-size: 1.2rem;
+    line-height: 2rem;
+  }
+
+  .button2 {
+    display: none;
+  }
+
+  .card-work {
+    display: none;
+  }
+
+  h3 {
+    width: 21rem;
+  }
+
+  .desktop-v .desktop-v3 {
+    background-image: url("./images/Img Placeholder.png");
+    background-repeat: no-repeat;
+    max-width: 750px;
+    margin: 0 auto;
+    width: 327px;
+    height: 450px;
+  }
+
+  .desktop-v1 .desktop-v4 {
+    background-image: url("./images/Img-Placeholder1.png");
+    background-repeat: no-repeat;
+    max-width: 750px;
+    margin: 0 auto;
+    width: 327px;
+    height: 450px;
+  }
+
+  .desktop-v2 .desktop-v5 {
+    background-image: url("./images/Img\ Placeholder\ \(2\).png");
+    background-repeat: no-repeat;
+    max-width: 750px;
+    margin: 0 auto;
+    width: 327px;
+    height: 450px;
+  }
+
+  .mini-projects:hover {
+    cursor: pointer;
+    border: 2px solid black;
+  }
+
+  .card-work-desktop:hover .mini-h3 {
+    display: none;
+  }
+
+  #about-me-description {
+    background-image: none;
+  }
+
+  h2.title {
+    text-align: left;
+  }
+
+  button#resume {
+    margin-top: 250px;
+  }
+
```

Comment:
- [ OPTIONAL] Kindly consider decreasing this margin-top, you must stick to the original design and use the same padding, margin, size, ...

- Figma design:

![good4](https://user-images.githubusercontent.com/79058364/161405089-9dbfe0ee-86c5-403c-8c6f-ac76e06a262c.png)

- Your design:

![bad4](https://user-images.githubusercontent.com/79058364/161405106-d9edb4ec-c786-4e12-bb8e-bfe24c57a10d.png)